### DISCUSSION TITLE
Ctrl+Shift+I and Ctrl+Shift+M no longer work

### DISCUSSION THREAD

**Discussion by SerialDreams** [Tyrrrz/DiscordChatExporter#787] [Q&A] [answered]
Trying to input those combos in the latest desktop ver. of Discord as of this comment (1/24/2022) does absolutely nothing and accessing the Application tab from the web ver. only provides empty subdirectories.

Am I doing something wrong?

**Comment by 96-LB** [comment, marked as answer]
The web version continues to work for me. Make sure that you're selecting this item on the Application tab:
![image](https://user-images.githubusercontent.com/21089343/150907074-e3ab2144-d845-4183-8641-cf228c4928f0.png)

**Reply by SerialDreams** [reply]
Much thanks!
For some reason it didn't immediately appear on refresh.

**Reply by Tyrrrz** [reply]
It's funny, because now Discord's own recommendations no longer apply: https://support.discord.com/hc/en-us/articles/115001239472-Troubleshooting-Console-Log-Errors

In any case, we should update the guide text to say that you need to open Discord in browser now because the existing method doesn't work. I'll create an issue.

**Comment by shelllter** [comment]
Any progress with this issue?

I'm using Browser based Discord on Windows 7 and Firefox, but as above, Ctrl-Shft-I and Ctrl-Shft-M only bring up a blank Applications page. The exact text (partially), is "No Service Workers found" (arg, you can't copy/paste it!). AND there is NO storage section displayed below on the let. Tried this several times, and waited a while.

The good news is the instructions like 
    dotnet DiscordChatExporter.Cli.dll guide
is working very nicely. Looking forward to resolving the token issue.

Thanks for all your hard work on this project!

**Reply by 96-LB** [reply]
Might be a browser issue. I haven't been able to replicate any problems on Windows 10 Chrome.

**Reply by 96-LB** [reply]
Update: a quick Google search seems to suggest that the local storage is stored on a different tab in Firefox. Try the 'Storage' tab, as shown [here](https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector).

**Reply by shelllter** [reply]
Yes, this seems to be it. 

( I have left my glasses in by my sleeping wife, so I'm having to squint really hard to read the text in the debugger/Storage. )

I see 'token' and 'tokens'. Both values are very "wide", I guess I was expecting 8 chars or less. 
Can you please confirm the 'token' will be approx 60 chars wide? 

Thanks very much for the quick reply!

**Reply by 96-LB** [reply]
Yes, that's normal. They're that long to guarantee that they're unique and unguessable.

**Reply by shelllter** [reply]
In the Browser version of Discord, the toggle for Developer Mode seems to now be under the Advanced section at the bottom of the settings page.

**Reply by shelllter** [reply]
Wu-hu, got my first channel downloaded! Thanks a bunch!

**Comment by HBIDamian** [comment]
According to `%localappdata%\Discord\app-1.0.9004\modules\discord_desktop_core-3\discord_desktop_core\core.asar` (decompile, then `\app\index.ts` Line 115-116), they've only disabled DevTools on the stable version of Discord.

```js
  const enableDevtoolsSetting = global.appSettings.get('DANGEROUS_ENABLE_DEVTOOLS_ONLY_ENABLE_IF_YOU_KNOW_WHAT_YOURE_DOING', false);
  const enableDevtools = buildInfo.releaseChannel === 'stable' ? enableDevtoolsSetting : true;
```

So `CTRL + SHIFT + I` should work on anything but Stable.
